test/bench/index.ts

```diff
@@ -0,0 +1,33 @@
+import * as BSON from '../register-bson.js';
+import { getStringDeserializationSuite } from './suites/string_deserialization';
+import { PerfSendData } from './util';
+import { writeFile } from 'fs';
+import { cpus, totalmem } from 'os';
+
+const hw = cpus();
+const platform = { name: hw[0].model, cores: hw.length, ram: `${totalmem() / 1024 ** 3}GiB` };
+
+const results: PerfSendData[][] = [];
+
+console.log(
+  [
+    `\n- cpu: ${platform.name}`,
+    `- cores: ${platform.cores}`,
+    `- os: ${process.platform}`,
+    `- ram: ${platform.ram}`
+  ].join('\n')
+);
+
+for (const suite of [getStringDeserializationSuite(BSON)]) {
```

Comment:
This is meant to collect all the suites we may end up adding to the `suites` directory

Reply:
It will be important for us to isolate these micro bench marks in individual nodejs processes, since this is the only one this is fine, can you leave a comment about that? Like "before adding more benches, either comment out the existing one or setup child process logic".